### PR TITLE
update stone specs

### PR DESCRIPTION
**Pull request by risforrob**
Doing this as a formal PR to for a second set of eyes.

### CODE REVIEW

File: common.stone [hunk 1]
```diff
@@ -28,21 +45,21 @@ union PathRoot
     member_home
         "Paths are relative to the authenticating team member's home
         directory. (This results in :field:`PathRootError.invalid' if the
-        user does not belong to a team.)"
+        user does not belong to a team.)."
```
Comment (posita) [inline common.stone]: Nit: Remove final period (cf. http://www.thepunctuationguide.com/parentheses.html).

File: common.stone [hunk 2]
```diff
@@ -28,21 +45,21 @@ union PathRoot
...
 
     team PathRootId
         "Paths are relative to the given team directory. (This results in
         :field:`PathRootError.invalid` if the user is not a member of
-        the team associated with that path root id.)"
+        the team associated with that path root id.)."
```
Comment (posita) [inline common.stone]: See above.

File: common.stone [hunk 3]
```diff
@@ -28,21 +45,21 @@ union PathRoot
...
 
     user_home
         "Paths are relative to the user's home directory. (This results in
-        :field:`PathRootError.invalid` if the belongs to a team.)"
+        :field:`PathRootError.invalid` if the belongs to a team.)."
```
Comment (posita) [inline common.stone]: See above.

File: common.stone [hunk 4]
```diff
@@ -28,21 +45,21 @@ union PathRoot
...
 
     namespace_id PathRootId
         "Paths are relative to given namespace id (This results in
         :field:`PathRootError.no_permission` if you don't have access
-        to this namespace.)"
+        to this namespace.)."
```
Comment (posita) [inline common.stone]: See above.

File: shared_links.stone [hunk 5]
```diff
@@ -298,20 +309,20 @@ union_closed SharedLinkSettingsError
         (for example, all attributes of the :type:`SharedLinkSettings` are empty,
         the requested visibility is :field:`RequestedVisibility.password` but the
         :field:`SharedLinkSettings.link_password` is missing, :field:`SharedLinkSettings.expires`
-        is set to the past, etc.)"
+        is set to the past, etc.)."
```
Comment (posita) [inline shared_links.stone]: See above.

File: common.stone (hunk 1 above)
Comment (risforrob) [reply common.stone]: I think this is a byproduct of some Linting jason added.

**Pull request merged**